chat-plugins/trivia.js

```diff
@@ -480,25 +481,18 @@ class Trivia extends Rooms.RoomGame {
 		let ladder = triviaData.ladder = [];
 		for (let i = 0; i < 3; i++) {
 			leaders.sort((a, b) => leaderboard[b][i] - leaderboard[a][i]);
-
-			let max = Infinity;
 			let rank = 0;
 			let rankIdx = i + 3;
 			for (let j = 0; j < leaders.length; j++) {
 				let leader = leaders[j];
-				let score = leaderboard[leader][i];
-				if (max !== score) {
-					if (!i && rank < 15) {
-						if (ladder[rank]) {
-							ladder[rank].push(leader);
-						} else {
-							ladder[rank] = [leader];
-						}
+				if (!i) {
```

Comment:
Don't change the algorithm for this for now. It's something I'll get around to rewriting later, since it's really inefficient